tests/foreman/api/test_subscription.py

```diff
@@ -338,40 +310,24 @@ def test_positive_candlepin_events_processed_by_stomp(
 
     :CaseImportance: High
     """
-    repo = target_sat.api.Repository(
-        product=target_sat.api.Product(organization=function_org).create()
-    ).create()
-    repo.sync()
-    ak = target_sat.api.ActivationKey(
-        content_view=function_org.default_content_view,
-        max_hosts=100,
-        organization=function_org,
-        environment=target_sat.api.LifecycleEnvironment(id=function_org.library.id),
-        auto_attach=True,
-    ).create()
-    result = rhel_contenthost.api_register(
-        target_sat,
-        organization=function_org,
-        activation_keys=[ak.name],
-    )
-    assert result.status == 0, f'Failed to register host: {result.stderr}'
-    host = target_sat.api.Host().search(query={'search': f'name={rhel_contenthost.hostname}'})
-    host_id = host[0].id
-    host_content = target_sat.api.Host(id=host_id).read_json()
-    assert host_content['subscription_status'] == 2
-    with function_entitlement_manifest as manifest:
-        target_sat.upload_manifest(function_org.id, manifest.content)
-    subscription = target_sat.api.Subscription(organization=function_org).search(
-        query={'search': f'name="{DEFAULT_SUBSCRIPTION_NAME}"'}
-    )[0]
-    target_sat.api.HostSubscription(host=host_id).add_subscriptions(
-        data={'subscriptions': [{'id': subscription.cp_id, 'quantity': 1}]}
-    )
-    host_content = target_sat.api.Host(id=host_id).read_json()
-    assert host_content['subscription_status'] == 0
-    response = target_sat.api.Ping().search_json()['services']['candlepin_events']
-    assert response['status'] == 'ok'
-    assert '0 Failed' in response['message']
+
+    # Function to parse candlepin events
+    def parse(events):
+        return {key: int(value) for value, key in re.findall(r'(\d+)\s(\w+)', events)}
+
+    manifester = Manifester(manifest_category=settings.manifest.entitlement)
```

Comment:
Why we are using entitlment here, can't we use golden_ticket?
```suggestion
    manifester = Manifester(manifest_category=settings.manifest.golden_ticket)
```
The purpose behind asking the question is that we are now using other manifest option and not entitlement manifest, so why not here?

Reply:
Good catch. Ill update it